xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+
+    <para>
+      &sudo; is a very flexible tool with extensive configuration.
+    </para>
+
+    <note>
+      <title>Locked out of sudo</title>
+      <para>
+        Should you lock yourself out of &sudo; by mistake, you can use <command>su -</command>
+        and the &rootuser; password to get a root shell.
+      </para>
+    </note>
+
+    <sect2 xml:id="sec.sudo.conf.edit">
+      <title>Editing the configuration files</title>
+      <para>
+        The main policy configuration file for &sudo; is <filename>/etc/sudoers</filename>.
+        As it is possible to lock yourself out of the system if there are any errors in that file, it is
+        strongly recommended to use <command>visudo</command> for editing.
+        It will prevent simultaneous changes to the opened file and check for syntax errors before saving.
+      </para>
+      <para>
+        Despite its name, you can also use editors other than vi by setting the <envar>EDITOR</envar>
+        environment variable, for example:
+      </para>
+      <screen>sudo EDITOR=<replaceable>/usr/bin/nano</replaceable> visudo</screen>
+
+      <para>
+        However, the <filename>/etc/sudoers</filename> file itself is supplied by the system packages and
+        modifications may break on updates. Therefore, it is recommended to put any custom configuration into
+        the <filename>/etc/sudoers.d/</filename> directory. Any file in there is automatically included.
+        To edit a file in that subdirectory, run:
+        </para>
+      <screen>sudo visudo -f /etc/sudoers.d/<replaceable>NAME</replaceable></screen>
+
+      <para>Or with <command>nano</command> as editor:</para>
+      <screen>sudo EDITOR=<replaceable>/usr/bin/nano</replaceable> visudo -f /etc/sudoers.d/<replaceable>NAME</replaceable></screen>
+
+      <note>
+        <title>Ignored files in <filename>/etc/sudoers.d</filename></title>
+        <para>
+          The <literal>#includedir</literal> command in <filename>/etc/sudoers</filename>,
+          used for <filename>/etc/sudoers.d</filename>, ignores files that end in <literal>~</literal>
+          or <emphasis>contain</emphasis> a <literal>.</literal>.
+        </para>
+      </note>
+
+      <para>For more info on the <command>visudo</command> command, run <command>man 8 visudo</command>.</para>
+    </sect2>
+
+    <sect2 xml:id="sec.sudo.conf.syntax">
+      <title>Basic sudoers syntax</title>
+      <para>
+        The most important syntax constructs for <filename>/etc/sudoers</filename>
+        and files under <filename>/etc/sudoers.d</filename> are:
+      </para>
+
+      <screen>
+# Everything on a line after a # gets ignored <co xml:id="co.sudo.syntax.comment"/>
+Defaults !insults # Disable the insults flag <co xml:id="co.sudo.syntax.flag"/>
+Defaults env_keep += "DISPLAY HOME" # Add DISPLAY and HOME to env_keep
+&exampleuser_plain; ALL = NOPASSWD: /usr/bin/frobnicate, PASSWD: /usr/bin/journalctl <co xml:id="co.sudo.syntax.rule"/>
```

Comment:
Not sure if "frobnicate" is a good name, though clearly a traditional choice... No good proposals though, except maybe using `<replaceable>COMMAND</replaceable>` instead.